maps/away/away_site/konyang/point_verdant/point_verdant.dm

```diff
@@ -6,6 +6,7 @@
 	suffixes = list("away_site/konyang/point_verdant/point_verdant-1.dmm","away_site/konyang/point_verdant/point_verdant-2.dmm","away_site/konyang/point_verdant/point_verdant-3.dmm")
 	spawn_weight = 1
 	spawn_cost = 1
+	template_flags = TEMPLATE_FLAG_SPAWN_GUARANTEED
```

Comment:
This needs to be merged after the change of the spawn guaranteed PR is merged?

Reply:
oops - good catch

Reply:
I forgot to remove the flag, it was there just for testing before I even put up the spawn guaranteed thing PR